internal/report/report.go

```diff
@@ -83,44 +78,6 @@ func processTemplate(w io.Writer, tmpl string, data interface{}) error {
 	return nil
 }
 
-func operatorInstallJsonReport(w io.Writer, ca options) error {
```

Comment:
Why not keep these, and just pass in the template data instead of exposing processTemplate?

Reply:
agreed 👍